### PR TITLE
login error, font style , validation fixed

### PR DESCRIPTION
**Pull request by rakeshkumar-verma**

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signin/clientlibs/less/signin.less [hunk 1]
```diff
@@ -9,6 +9,7 @@
 @rememberms-font-color:#7c7c7c;
 
 .cmp-login {
+	font-family: Arial, Helvetica, sans-serif;
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signin/clientlibs/less/signin.less]: @rakeshkumar-verma : we are using the font family with san-serif as priority in button.css . why have we changed priority here ?

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/less/button.less [hunk 2]
```diff
@@ -230,6 +231,7 @@ link-primary {
 }
 
 .logout {
+	font-family: Arial, Helvetica, sans-serif;
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/less/button.less]: @rakeshkumar-verma : we are using the font family with san-serif as priority in button.css . why have we changed priority here ?

**Review by rishabh-behl — COMMENTED** [review]
@rakeshkumar-verma : Pl check comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/less/button.less (hunk 2 above)
Comment (umeshv12) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/less/button.less]: Updated back to san-serif

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signin/clientlibs/less/signin.less (hunk 1 above)
Comment (umeshv12) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signin/clientlibs/less/signin.less]: Updated back to san-serif

Comment (rakeshkumar-verma) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signin/clientlibs/less/signin.less]: @rishabh-behl  updated to  sans-serif;

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/less/button.less (hunk 2 above)
Comment (rakeshkumar-verma) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/less/button.less]: @rishabh-behl   updated font family as  sans-serif;

**Pull request merged**